fe1-web/components/TextBlock.tsx

```diff
@@ -15,20 +15,24 @@ const styles = StyleSheet.create({
 const TextBlock = (props: IPropTypes) => {
   const { text } = props;
   const { bold } = props;
+  const { visibility } = props;
 
-  return (
-    <Text style={(bold) ? styles.textBold : styles.textStandard}>{text}</Text>
-  );
+  return (visibility)
+    ? (
+      <Text style={(bold) ? styles.textBold : styles.textStandard}>{text}</Text>
+    ) : null;
 };
```

Comment:
It is good practice to take a look at [SonarCloud report](https://sonarcloud.io/project/issues?id=dedis_student_21_pop_fe1&pullRequest=259&resolved=false&types=CODE_SMELL). Here, it gives you a "major warning"! ^^'